sdk/appservice/mgmt/pom.xml

```diff
@@ -87,6 +87,16 @@
       <version>3.3</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core-http-netty</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-identity</artifactId>
```

Comment:
Why we need add azure-identity in test?

Reply:
I have removed it from compile scope, so it needs to be added in test scope.

Reply:
Got it.